<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FundDialog</class>
 <widget class="QDialog" name="FundDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>384</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbFundTransactionDetails">
     <property name="title">
      <string>Fund Transaction Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblCode">
        <property name="text">
         <string>Code :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leCode">
        <property name="maxLength">
         <number>5</number>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leName">
        <property name="maxLength">
         <number>250</number>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblPrice">
        <property name="text">
         <string>Price :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="dsbPrice">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblCount">
        <property name="text">
         <string>Count :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="sbCount">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblOrderDate">
        <property name="text">
         <string>Order Date :</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDateEdit" name="deOrderDate">
        <property name="maximumDate">
         <date>
          <year>2250</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblOperationDate">
        <property name="text">
         <string>Operation Date :</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDateEdit" name="deOperationDate">
        <property name="maximumDate">
         <date>
          <year>2250</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblGoalPrice">
        <property name="text">
         <string>Goal Price :</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="dsbGoalPrice">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblOperationType">
        <property name="text">
         <string>Operation Type :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbOperationType"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbTransactionApproval">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlStatusBar"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbOperationType</tabstop>
  <tabstop>leCode</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>dsbPrice</tabstop>
  <tabstop>sbCount</tabstop>
  <tabstop>deOrderDate</tabstop>
  <tabstop>deOperationDate</tabstop>
  <tabstop>dsbGoalPrice</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bbTransactionApproval</sender>
   <signal>accepted()</signal>
   <receiver>FundDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbTransactionApproval</sender>
   <signal>rejected()</signal>
   <receiver>FundDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
